src/read/unit.rs

```diff
@@ -4751,6 +4756,27 @@ mod tests {
         test_parse_attribute(&buf, bytes_written, &unit, form, value);
     }
 
+    #[test]
+    fn test_parse_attribute_indirect_implicit_const() {
+        let encoding = Encoding {
+            format: Format::Dwarf32,
+            version: 4,
+            address_size: 4,
+        };
+        let mut buf = [0; 100];
+        let mut writable = &mut buf[..];
+        leb128::write::unsigned(&mut writable, constants::DW_FORM_implicit_const.0.into())
```

Comment:
```suggestion
        leb128::write::unsigned(&mut &mut buf[..], constants::DW_FORM_implicit_const.0.into())
```

Reply:
Yep, thanks. This is old copy+paste test code though, so going to leave it how it is for consistency.